.editorconfig

```diff
@@ -0,0 +1,75 @@
+# EditorConfig (is awesome!; ref: http://EditorConfig.org)
+# ref: [EditorConfig ~ Level-0 Coding Standard](https://themightyprogrammer.dev/article/editor-config) @@ <https://archive.is/5wK2a>
+# v2022.10.18; rivy
+
+# spell-checker:ignore (words) akefile makefile makefiles MSVC POSIX VCproj
+
+# * top-most EditorConfig file
+root = true
+
+[*]
+# default ~ utf-8, unix-style newlines with a newline ending every file, 4 space indentation (and tab width)
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+max_line_length = 100
+smart_tabs = unset  # ## unstable; ref: <https://github.com/editorconfig/editorconfig/issues/323>
+tab_width = unset # default to 'indent_size'
+trim_trailing_whitespace = true
+
+[{[Mm]akefile{,.*},*.{mak,mk,[Mm][Aa][Kk],[Mm][Kk]},[Gg][Nn][Uu]makefile}]
+# makefiles ~ TAB-style indentation
+indent_style = tab
+
+[*.{bash,sh}]
+# Linux/POSIX shell scripts
+indent_size = 4
+indent_style = space
+
+[*.{bat,cmd,[Bb][Aa][Tt],[Cc][Mm][Dd]}]
+# BAT/CMD ~ DOS/Win requires BAT/CMD files to have CRLF EOLNs
+end_of_line = crlf
+
+[*.{c,cc,cpp,cxx,h,hh,hpp,hxx,[Cc],[Cc][Cc],[Cc][Pp][Pp],[Cc][Xx][Xx],[Hh],[Hh][Hh],[Hh][Pp][Pp],[Hh][Xx][Xx]}]
```

Comment:
why bother to add the configuration for languages we won't use?

Reply:
It's really for development convenience. I try to keep one commented/"versioned" file for use where-ever I'm developing.
I can test vs my common config for any updates; it helps me keep up with changes and use the same practices throughout my projects.

I think it's useful without any real downside. Do you disagree/object?

In review of the file just now, I notice that I don't have TOML. I'm going to "DRAFT" for the moment until I add TOML, probably later today.

Reply:
I would prefer to keep only relevant information